package.json

```diff
@@ -96,7 +96,7 @@
   },
   "dependencies": {
     "@elastic/datemath": "link:packages/elastic-datemath",
-    "@elastic/elasticsearch": "npm:@elastic/elasticsearch-canary@^8.0.0-canary.3",
+    "@elastic/elasticsearch": "github:elastic/elasticsearch-js#type-definitions",
```

Comment:
Any context on this change?  Plans for version management?

Reply:
It's a temporary measurement. We are going to release another canary package when this PR is approved.